packages/unenv-preset/tests/worker/index.ts

```diff
@@ -10,6 +10,7 @@ export const TESTS = {
 	testPath,
 	testDns,
 	testTimers,
```

Comment:
Should there also be a `testTimersPromise`?

Reply:
Ideally yes